src/bot.py

```
@@ -6,784 +6,740 @@
-            continue
+        start_time = time.perf_counter()
+        config = self.config
 
-        formatted_text = node.text[:max_text]
+        # 1. Configuration Resolution
+        provider_slash_model = config.chat.channel_models.get(message.channel.id, config.chat.default_model)
+        try:
+            provider, model = provider_slash_model.removesuffix(":vision").split("/", 1)
+            provider_config = config.llm.providers[provider]
+            openai_client = self.get_openai_client(provider_config)
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)